.compose.env

```diff
@@ -0,0 +1,6 @@
+# Determines how the dev environment should run. Set to insights for developing
```

Comment:
Please rename it to `.compose.env.example`, since `.compose.env` is added to `.gitignore`